config/filterMap.py

```diff
@@ -30,17 +30,25 @@
         ("HSC-Y", "y"),
         # Narrow bands
         ('NB0387', 'g'),
+        ('NB0391', 'g'),
+        ('NB0395', 'g'),
+        ('NB0430', 'g'),
         ('NB0468', 'g'),
+        ('NB0497', 'g'),
+        ('NB0506', 'g'),
         ('NB0515', 'g'),
         ('NB0527', 'g'),
         ('NB0656', 'r'),
         ('NB0718', 'i'),
         ('NB0816', 'i'),
+        ('NB0871', 'i'),
         ('NB0921', 'z'),
         ('NB0926', 'z'),
         ('NB0973', 'y'),
         ('NB1010', 'y'),
         # Intermediate bands
         ('IB0945', 'z'),
+        # Extended bands
+        ('EB-gri', 'r'),
```

Comment:
For reasons I'm not entirely clear about, but which I might have inherited from the gen2 code, the metadata translator upper cases the filter name so this has to be EB-GRI for the ingest to work.

Reply:
Not sure what this config is used for but should this by "gri" band to match the definition below?

Reply:
It selects which band to use for photometric calibration. This says "for filter EB-GRI, calibrate against r".